spec/support/page_objects/providers/index.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Providers
+    class Index < PageObjects::Base
+      set_url "/providers"
+
+      element :add_provider_link, "a", text: "Add a provider"
+
+      elements :provider_data, ".provider-card"
+
+      elements :provider_name, ".provider-card .govuk-link"
```

Comment:
It looks :tropical_fish:  (uped `.provider-card` )

Can we use
https://rdoc.info/gems/site_prism/SitePrism/Section